Client/game_sa/CFileLoaderSA.cpp

```diff
@@ -159,3 +144,87 @@ RpAtomic* CFileLoader_SetRelatedModelInfoCB(RpAtomic* atomic, SRelatedModelInfo*
     }
     return atomic;
 }
+
+bool IsMatrixNeeed(SFileObjectInstance* pInst, CEntitySAInterface* pEntity)
+{
+    const bool bHasRotation = std::fabs(pInst->rotation.fX) > 0.050000001f || std::fabs(pInst->rotation.fY) > 0.050000001f ||
+        ((pInst->interiorID & 0xFF00) >> 8) & 2;
+    if (!bHasRotation) {
+        return false;
+    }
```

Comment:
```suggestion
    if (!(((pInst->interiorID & 0xFF00) >> 8) & 2))
        return false;

    if (std::fabs(pInst->rotation.fX) <= 0.05f && std::fabs(pInst->rotation.fY) <= 0.05f)
        return false;
```
Could you please explain the first `if`?

Reply:
Okay, just looked at the original GTA code at `0x5381E0`.
Just add a comment it's from GTA code.